dg_test.go

```diff
@@ -182,6 +182,22 @@ func TestDirectedGraph_OneAndDependencyConnectDependency(t *testing.T) {
 		})
 }
 
+func TestDirectedGraph_ResolvingSingleNode(t *testing.T) {
+	// This test makes sure that it's only dependent nodes that get marked as ready.
+	// Since there are no dependencies connected here, there should be no nodes marked as ready.
```

Comment:
I would enhance the second sentence along these lines:
> Since there are no dependencies between nodes here, marking nodes as resolved should not cause any nodes to be placed on the ready list.

Reply:
Done.